web/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "ades-webapp",
+  "private": true,
+  "devDependencies": {
+    "http-server": "14.1.1"
```

Comment:
[Configure Dependabot](https://github.com/ericcornelissen/ades/blob/aea637182815d57705e13c217dc00d6d610c2334/.github/dependabot.yml) to keep these dependencies up to date.